angular-intro-project/src/app/search-and-add-section/search-and-add-section.component.html

```diff
@@ -1,3 +1,8 @@
-<app-search-course-input></app-search-course-input>
-<app-search-course-button></app-search-course-button>
+<!-- <app-search-course-input></app-search-course-input>
+<app-search-course-button></app-search-course-button> -->
+<div class='d-flex w-75 mt-2'>
+  <input class='search-course-input' [(ngModel)]='inputValue'>
+  <button class='search-course-button ml-2' (click)="onSearchCourseClick()">Search</button>
+</div>
+
 <app-add-course-button></app-add-course-button>
```

Comment:
I think we don't need to combine search section with add button into particular component, it seems over engineering for me.
So, I would rename this component into SearchSection for example and would move AddCourseButton component on the same level as SearchSection.